src/main/resources/fileTemplates/config.jelly.ft

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright DATE, Issuer.....
+
+Licensed under ...
+
+@Author
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" >
+    <f:section title="${%Put Section Name Here}">
+
+        <!-- see https://www.jenkins.io/doc/developer/forms/ for help -->
+
```

Comment:
```suggestion
        <!-- see https://www.jenkins.io/doc/developer/forms/ for help -->
        #[[$END$]]#
```
Not 100%, but I think this specifies where caret should go after the file is created from the template.